### ISSUE TITLE
Can't open any notebook

### ISSUE THREAD

**Issue by aleksey-uvarov** [millejoh/emacs-ipython-notebook#362]
## Check list

- [*] Read the "Avoiding Common Emacs Traps" section in
  https://github.com/millejoh/emacs-ipython-notebook/blob/master/CONTRIBUTING.md
- [*] Does IPython works from the web browser?
- [*] Confirm the problem was not due to badly compiled
  files by removing all the `*.elc` files from source directory of EIN and
  its dependencies.
- [*?] Confirm that the libraries are loaded as expected and are at the required version.
  (You can check the location in the "System info" section below)

## Description of the problem you have

Jupyter server starts normally, but when I try to open any notebook by
pressing the "Open" button, nothing happens. This issue resembles the
issue #106 from tkf's repository:
https://github.com/tkf/emacs-ipython-notebook/issues/106

However, that issue relates to some older version of EIN, so its ultimate fate is unknown to me

## Your EIN configuration (in .emacs.d/init.el or somewhere else)

none

## Your IPython configuration

1. What is your IPython version? (run `ipython --version`):

6.5.0

2. How do you start IPython? (e.g., `ipython notebook --port 9999`):

jupyter notebook

3. What is your IPython notebook port number or URL?:

8888

## Additional information (if any)

The locations of the packages are correct, but I'm not sure if the
"version nil" is how it is supposed to look like

## System info:

```cl
("EIN system info"
 :emacs-version
 "GNU Emacs 25.2.2 (x86_64-pc-linux-gnu, GTK+ Version 3.22.21)
 of 2017-09-22, modified by Debian"
 :emacs-bzr-version nil
 :window-system x
 :emacs-variant nil
 :os
 (:uname
  "Linux alexey-desktop 4.15.0-36-generic #39-Ubuntu SMP Mon Sep 24 16:19:09 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
"
  :lsb-release
  "No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 18.04.1 LTS
Release:	18.04
Codename:	bionic
")
 :image-types
 (svg imagemagick png gif tiff jpeg xpm xbm pbm postscript)
 :image-types-available
 (svg imagemagick png gif tiff jpeg xpm xbm pbm postscript)
 :request
 (:backend url-retrieve)
 :ein
 (:version "0.14.0"
	   :source-dir "/home/alexey/.emacs.d/elpa/ein-0.14.1/")
 :lib
 ((:name "websocket"
	 :path "~/.emacs.d/elpa/websocket-1.10/websocket.el"
	 :featurep t
	 :version-var websocket-version
	 :version "1.9")
  (:name "request"
	 :path "~/.emacs.d/elpa/request-0.3.0/request.el"
	 :featurep t
	 :version-var request-version
	 :version "0.3.0")
  (:name "auto-complete"
	 :path "~/.emacs.d/elpa/auto-complete-1.5.1/auto-complete.el"
	 :featurep t
	 :version-var nil
	 :version nil)
  (:name "auto-complete"
	 :path "~/.emacs.d/elpa/auto-complete-1.5.1/auto-complete.el"
	 :featurep t
	 :version-var nil
	 :version nil)
  (:name "popup"
	 :path "~/.emacs.d/elpa/popup-0.5.3/popup.el"
	 :featurep t
	 :version-var popup-version
	 :version "0.5.3")
  (:name "python"
	 :path "/usr/share/emacs/25.2/lisp/progmodes/python.elc"
	 :featurep nil
	 :version-var nil
	 :version nil)))
```

**Comment by dickmao** [comment]
Thank you for first doing a search in Issues!
Your version says 0.14.0.  Have you reinstalled the ein package from melpa in the last 24 hours?

**Comment by millejoh** [comment]
You might also want to try the curl backend instead of url-retrieve. For some time curl has tended to be the more robust option.

**Comment by aleksey-uvarov** [comment]
>Have you reinstalled the ein package from melpa in the last 24 hours?

I installed ein from melpa for the first and (so far) only time on the same day I opened the issue

>You might also want to try the curl backend instead of url-retrieve.

I'm somewhat new to emacs (and totally new to ein), how do I do this? 

Thanks for the replies anyway! I'll be able to test the advice proposed no sooner than tomorrow morning (Moscow time)

**Comment by millejoh** [comment]
Are you downloading from MELPA stable? I really do need to put my foot down and release a 0.14.2 (though really i think we are at 0.15.0 at this point with all of @dickmao's great work).

Changing the request backend is fairly easy, and as is the case with emacs there is more than one way to do it. The method I would recommend is the following:

Type `M-x customize-variable RET` then `request-backend RET`. That will bring up the customization buffer. Click on the "Value Menu" button and then select "cURL Backend".

Hope that helps some - let us know how things work out.

**Comment by aleksey-uvarov** [comment]
1. Uninstalled and then reinstalled from MELPA

Apparently I can now open new notebooks and access those I just opened using EIN. Notebooks created previously using the web client are still unopenable.

2. Changed backend to cURL.

Didn't work at first, but seems like I just didn't have curl installed. After installing curl, it starts fine and opens any notebooks, but when I try to evaluate cells in a notebook not created within EIN, it says this:

`ein: [warn] Kernel is not ready yet! (or closed already.)`
